packages/medusa/src/services/cart.ts

```diff
@@ -536,6 +573,17 @@ class CartService extends TransactionBaseService<CartService> {
           ],
         })
 
+        const isSalesChannelAssociationValid = validateSalesChannels
```

Comment:
suggestion: If we rename the method as suggested above, then it can become `isValid`

Reply:
in that case, yes, but I think we should keep the method as is until additional validations are needed. (also I was thinking extra hard to come up with a descriptive name here and would be sad if all that was in vain :/)

Reply:
Ahah i understand, you coukd have name it doesLineItemBelongsToSalesChannel 😉